EssentialFeed/EssentialFeediOS/Comments UI/FeedCommentsViewController.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  FeedCommentsViewController.swift
+//  EssentialFeediOS
+//
+//  Created by Danil Vassyakin on 3/31/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import UIKit
+import EssentialFeed
+
+public protocol FeedCommentsViewViewControllerDelegate {
+	func didRequestCommentsRefresh()
+}
+
+public final class FeedCommentsViewController: UITableViewController, FeedImageCommentView, FeedImageCommentLoadingView, FeedImageCommentErrorView {
+
+	public var delegate: FeedCommentsViewViewControllerDelegate?
+	@IBOutlet weak var errorView: ErrorView!
```

Comment:
The setter can be private  to prevent clients from mutating the property:

```suggestion
	@IBOutlet private(set) weak var errorView: ErrorView!
```